.github/workflows/release_next.yml

```diff
@@ -171,12 +176,26 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-node@v3
         with:
-          python-version: 3.x
-      - run: pip install -r docs/pip_requirements.txt
-      - run: mkdocs gh-deploy --force -m "[ci skip]"
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm install --frozen-lockfile
         working-directory: docs
+        
+      - name: Test build docs
+        run: npm run build
+        working-directory: docs
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        working-directory: docs
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./build
+          user_name: github-actions[bot]
+          user_email: 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
What is this email address referring to?

Reply:
It's a GitHub bot that will deploy the build to `gh-pages`. Reference: https://github.com/actions/checkout/issues/13#issuecomment-724415212

Reply:
Cool. Let's add a comment for future reference